mmv1/products/biglake/Table.yaml

```diff
@@ -0,0 +1,133 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Resource
+name: "Table"
+description: |
+  Represents a table.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    "Manage open source metadata with BigLake Metastore": "https://cloud.google.com/bigquery/docs/manage-open-source-metadata#create_tables"
+  api: "https://cloud.google.com/bigquery/docs/reference/biglake/rest/v1/projects.locations.catalogs.databases.tables"
+base_url: "{{database}}/tables"
+self_link: "{{database}}/tables/{{name}}"
+create_url: "{{database}}/tables?tableId={{name}}"
+id_format: "{{database}}/tables/{{name}}"
+import_format: ["{{%database}}/tables/{{name}}"]
+update_verb: :PATCH
+update_mask: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "biglake_table"
+    primary_resource_id: "table"
+    vars:
+      name: "my_table"
+      catalog: "my_catalog"
+      database: "my_database"
+      bucket: "my_bucket"
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "name"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Output only. The name of the Table. Format:
+      projects/{project_id_or_number}/locations/{locationId}/catalogs/{catalogId}/databases/{databaseId}/tables/{tableId}
+  - !ruby/object:Api::Type::String
+    name: "database"
```

Comment:
just want to double-check that this is mutable? Changing a parent field usually requires deleting and recreating a resource.

Reply:
done.
marked immutable